src/lib/graph.js

```diff
@@ -23,12 +23,13 @@ const drawBase = (points) => {
 
 export const drawGraph = (values, { drawMean = false, fillLast = false } = {}) => {
   const maxValue = values.reduce((max, { value }) => (value > max ? value : max), 0)
-  const sum = values.reduce((acc, { value }) => acc + value, 0)
-  const mean = sum / values.length
+  const usableValues = values.filter(({ value }) => value !== Number.NEGATIVE_INFINITY)
+  const sum = usableValues.reduce((acc, { value }) => acc + value, 0)
+  const mean = sum / usableValues.length
```

Comment:
This change ensures, that the missing value does not have an effect on the mean value. So we filter the `-Infinity` out